src/pimp_mixer_arm.S

```diff
@@ -214,13 +213,14 @@ pimp_mixer_mix_samples:
 	ldr r4, =0x4000208 @ load address of REG_IME
 	ldr r5, .ime_store @ stash for later
```

Comment:
I guess this comment should have said "restore value"

Reply:
This was just a note to myself, though. I'm not asking you to fix the comment ;)